lib/zosung.js

```diff
@@ -10,8 +10,9 @@ function nextSeq(entity) {
 
 function messagesGet(entity, seq) {
     const info = entity.irMessageInfo;
-    if (info.seq!=seq) {
-        throw new Error(`Unexpected sequence value (expected: ${info.seq} current: ${seq}).`);
+    const expected = info?.seq || 0;
```

Comment:
The `?` operator is not supported, can you update this?

Reply:
Sorry, missed the lint script, fixed!

Just curious, what's the intended target of this package? Noticed that it lacks an `engines` field and nodejs 14 (minimum supported by `zigbee2mqtt`) does support the operator.